index.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <link rel="stylesheet" href="/css/styles.css" />
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>quizz app</title>
```

Comment:
I found a typo. Or is this your apps individual name?

Reply:
what do you mean ? the title ? quiz app is the title !

Reply:
ahhh you mean quiZZ...let change it